src/workloads/execution/ColumnStoreIndex.yml

```diff
@@ -302,51 +372,84 @@ TopTenantsByNoOfEventsQuery: &TopTenantsByNoOfEventsQuery
       {"$project": {"color": "$__alias_0", "x": "$__alias_2", "y": "$__alias_1", "_id": 0}},
       {"$limit": 50000}
     ]
-  cursor: {batchSize: 50000}
+  cursor: {}
 
 # Defines test operations.
 TestOperations: &TestOperations
 - OperationMetricsName: CountPerActionQuery
   OperationName: RunCommand
-  OperationCommand: *CountPerActionQuery
+  OperationCommand:
```

Comment:
This doesn't look like ```Fix use of ISODate in column store genny workload``` Should this be in a separate PR, or should we update the ticket? 

Also, could you explain in a comment why we are using explain here.

Reply:
According to the description, it meant working around the issue. Updated the title of the ticket. Also added the following comment.

```
# Here we use explain with verbosity = execustionStats instead of using plain aggregate command
# so that aggregate pipeline runs to completion but yet we don't need to exhaust cursor. As of now
# we don't have an easy way to exhaust cursor.
```